lib/adapters/QUnitAdapter.js

```diff
@@ -13,21 +13,72 @@ export default class QUnitAdapter extends EventEmitter {
     QUnit.moduleDone(this.onModuleDone.bind(this))
     QUnit.done(this.onDone.bind(this))
 
-    this.idDataMap = {}
+    this.testsDetails = {}
   }
 
   convertModule (qunitModule) {
     var suiteName = qunitModule.name
     var tests = []
+
     for (let qunitTest of qunitModule.tests) {
-      let test = new Test(qunitTest.name, suiteName)
-      tests.push(test)
-      this.idDataMap[qunitTest.testId] = test
+      qunitTest.testEnd = qunitModule.suiteEnd
```

Comment:
I don't understand what you're talking about. There are no `emit` calls in `convertTest`